timesketch/api/v1/resources/sigma.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Sigma resources for version 1 of the Timesketch API."""
+
+import logging
+
+from flask import abort
+from flask import jsonify
+from flask_restful import Resource
+from flask_login import login_required
+from flask_login import current_user
+
+import timesketch.lib.ts_sigma as ts_sigma_lib
+
+from timesketch.api.v1 import resources
+from timesketch.lib.definitions import HTTP_STATUS_CODE_NOT_FOUND
+
+logger = logging.getLogger('timesketch.api.sigma')
+
+class SigmaListResource(resources.ResourceMixin, Resource):
+    """Resource to get list of users."""
+
+    @login_required
+    def get(self):
+        """Handles GET request to the resource.
+
+        Returns:
+            Dict of sigma rules
+        """
+        sigma_rules = []
+
+        try:
+            _RULES_PATH = ts_sigma_lib.get_sigma_rules_path()
+
+        except ValueError as err:
+            logger.error("OS error: {0}".format(err))
+            abort(
+                HTTP_STATUS_CODE_NOT_FOUND,
+                err)
```

Comment:
and I would prefer a better text here in the abort statement, since htis is what is presented to the end user, rather than just a bare bone stacktrace....

Reply:
yep, easy fix, done